src/main/java/de/retest/web/selenium/AutocheckingRecheckDriver.java

```diff
@@ -0,0 +1,93 @@
+package de.retest.web.selenium;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import de.retest.web.RecheckWebImpl;
+
+public class AutocheckingRecheckDriver extends RecheckDriver {
```

Comment:
Aren't "auto" and "checking" two words, i.e., `AutoCheckingRecheckDriver`?

Reply:
Good question. I'd argument that it's not a checking recheck driver that is auto, but an auto-checking recheck driver. See more examples from https://www.grammarly.com/blog/hyphen/

> It’s recommended you don’t take down any load bearing walls when renovating.
>
> In this sentence, it sounds like you shouldn’t take down any load that is holding up a wall. A hyphen should be inserted between load and bearing to make it clear that we’re talking about walls that are bearing a load.
> 
> It’s recommended you don’t take down any load-bearing walls when renovating.
> This rock-hard cake is absolutely impossible to eat.
> We’re looking for a dog-friendly hotel.
>
> You should use the auto-checking driver when you don't want to be bothered with checking at all.